schema.graphql

```diff
@@ -233,3 +233,12 @@ type Solution @entity {
   txHash: Bytes!
   txLogIndex: BigInt!
 }
+
+type FleetDeployed @entity {
+  id: ID! # txHash + id
+  owner: Bytes!
+  fleet: [Bytes!]
+
+  # Transaction
+  txHash: Bytes!
```

Comment:
Look in other entities, it's nice to have audit times too

Reply:
Actually, the deployment of fleet-safe contracts will not be relevant at all. Only the placement of the orders will have any meaning in all use cases, which I can imagine.